website/www/site/content/en/documentation/io/built-in/google-bigquery.md

```diff
@@ -323,6 +323,11 @@ in the following example:
 {{< code_sample "sdks/python/apache_beam/examples/snippets/snippets.py" model_bigqueryio_read_query_std_sql >}}
 {{< /highlight >}}
 
+{{< paragraph class="language-java" >}}
+#### Query execution project
+By default the pipeline executes the query in the Google Cloud project associated with the pipeline (in case of the Dataflow runner it's the project where the pipeline runs). There are cases where the query execution project should be different from the pipeline project. You can define the query execution project by setting the pipeline option "[bigQueryProject](https://beam.apache.org/releases/javadoc/current/org/apache/beam/sdk/io/gcp/bigquery/BigQueryOptions.html#getBigQueryProject--)" to the desired Google Cloud project id.
+{{< /paragraph >}}
+
```

Comment:
```suggestion

{{< paragraph class="language-python" >}}
The SDK for Python does not support the query execution project option.
{{< /paragraph >}}

```

Reply:
How about this? Looks like the Python/Java Tab does not support adapting whole (sub)section. Currently even switched to Python Tab this paragraph is still shown.
![image](https://user-images.githubusercontent.com/8010435/219166163-f48e5754-c162-447f-b7e8-d79088d6d4b7.png)

Reply:
I don't feel comfortable adding adding "Python doesn't support it". I spent some time reading the logic of Python's IO and there is potentially a way to do it; but it requires testing or someone with the deeper knowledge of the transform needs to provide details. How about "If you use Java SDK, you can define the query execution project ...."?

Reply:
Understand. That sounds good to me. This is just a problem of the website rendering (if a section has only java content marked, it will always show up even though the current tab is python). Adding "If you use Java SDK," also clarifies it.